homeassistant/components/recorder/statistics.py

```diff
@@ -418,7 +495,13 @@ def _compile_hourly_statistics(session: Session, start: datetime) -> None:
 
     if stats:
         for stat in stats:
-            metadata_id, _mean, _min, _max = stat
+            metadata_id, _min, _max, _mean, _mean_type = stat
+            if (
+                try_parse_enum(StatisticMeanType, _mean_type)
+                is StatisticMeanType.CIRCULAR
+            ):
+                # Normalize the circular mean to be in the range [0, 360)
+                _mean = _mean % 360
```

Comment:
Why are the averages not stored modulo 360 in the database? Where does that happen?